Lesson11_Task01.py

```diff
@@ -0,0 +1,33 @@
+"""
+Lesson11 Task01.
+
+write generator function that has as input some range value and chunk_size
+produces output as lists with len == chunk size
+Example:
+Call:  chunk_generator(range(11), chunk_size=3) ->
+Output:  [0, 1, 2]
+         [3, 4, 5]
+         [6, 7, 8]
+         [9, 10]
+"""
+
+
+def chunk_generator(range_value, chunk_size):
+    """Create lists with chunks size from a given range."""
+    iterator = iter(range_value)
+    while True:
```

Comment:
In general Interesting solution with iterator and StopIteration handling ... but a little bit complicated.
As advanced please examine next example:

```
def chunks_generator(range_val, chunk_size=3):
    _range = range_val
    while _range:
        yield list(_range[:chunk_size])
        _range = _range[chunk_size:]


print(*[chunk for chunk in chunks_generator(range(23), chunk_size=5)], sep='\n')
```